cirq-core/cirq/qis/noise_utils_test.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2021 The Cirq Developers
```

Comment:
Same comment

Reply:
I don't know. I'm not creating new content. it's the same file. @pavoljuhas what do you think?

Reply:
I think either way is fine in this case, feel free to leave it as is.
For new module with new code we should use current year.